pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchFromGitHub, glibcLocales
+, cmake, eigen, openmpi, gtest, python3
```

Comment:
If you're not currently using some of the arguments, should they still be there? I feel you have listed the packages needed in your comment

Reply:
removed unneeded, they get pulled down as part of the submodules.